src/rust/Dockerfile

```diff
@@ -263,15 +263,24 @@ COPY --from=build /outputs/plugin-registry /
 COPY --from=build /nomad/nomad /bin
 ENTRYPOINT ["/plugin-registry"]
 
-# plugin-bootstrap
+##### plugin-bootstrap
 FROM rust-dist AS plugin-bootstrap-deploy
 
 # hadolint ignore=DL3022
 COPY --from=build /outputs/plugin-bootstrap /
 ENTRYPOINT ["/plugin-bootstrap"]
 
+##### plugin-bootstrap-init.
+# Not "-deploy" since we copy it to dist/.
+FROM scratch AS plugin-bootstrap-init-output
 
-# node-identifier
+# hadolint ignore=DL3022
+WORKDIR /plugin-bootstrap-init
+COPY --from=build /outputs/plugin-bootstrap-init .
+COPY rust/plugin-bootstrap/plugin-bootstrap-init.service .
+COPY rust/plugin-bootstrap/plugin.service .
```

Comment:
Just to clarify: we're copying these `*.service` files from the repository, through Docker, and then back out to the `dist` directory in the repository?

I can see the utility of keeping all that logic consolidated in one place, but it might be worth acknowledging the oddness of the situation in a comment.

(Might also be worth simplifying the last two lines to `COPY rust/plugin-bootstrap/*.service .`)